vllm/config/model.py

```diff
@@ -1516,6 +1516,12 @@ def get_mamba_chunk_size(self) -> int | None:
         if chunk_size is None:
             # used by e.g. Mamba2, NemotronH, Zamba
             chunk_size = getattr(self.hf_text_config, "chunk_size", None)
+
+        # Since Mamba1 does not have a chunk notion,
+        # use 2048 to align with the kernel chunking.
```

Comment:
Which "kernel chunking" are you referring to here? How is that different to a "chunk notion" like we have in Mamba2?

Reply:
OK - so it's like a runtime parameter rather that something fixed by the model config?